qtpy/tests/test_qtgui.py

```diff
@@ -12,12 +12,16 @@
     sys.platform.startswith('linux') and not_using_conda(),
     reason="Fatal Python error: Aborted on Linux CI when not using conda")
 def test_qfontmetrics_width(qtbot):
-    """Test QFontMetrics width"""
+    """Test QFontMetrics and QFontMetricsF width"""
     assert QtGui.QFontMetrics.width is not None
+    assert QtGui.QFontMetricsF.width is not None
     font = QtGui.QFont("times", 24)
     font_metrics = QtGui.QFontMetrics(font)
+    font_metricsF = QtGui.QFontMetricsF(font)
     width = font_metrics.width("Test")
+    widthF = font_metricsF.width("Test")
     assert width in range(40, 62)
+    assert widthF == width
```

Comment:
Seems like this is causing the test to file since `widthF` is a floating point number while `width` is an integer so this end up being a validation like `54.484375 == 54` I would suggest to use something like the assert above:

```suggestion
    assert  40 <= widthF <= 60
```

Reply:
Actually that test was passing but your point is solid.